src/common/addlayers/UnifiedSearchDirective.js

```diff
@@ -728,6 +730,10 @@
                     layer.name = layer.typename;
                     LayersService.addLayer(layer, server.id, server);
                   });
```

Comment:
Like 717 shoudl change to `if (server_to_use !== null)`

Reply:
Gotcha, will update